content/md/en/docs/reference/how-to-guides/basics/import-a-pallet.md

```diff
@@ -106,5 +106,5 @@ std = [
 ## Related material
 
 - [Timestamp Pallet associated types](https://paritytech.github.io/substrate/master/pallet_timestamp/index.html)
-- [FRAME `pallet-timestamp`](https://crates.io/crates/pallet-timestamp)
+- [FRAME `pallet-timestamp`](https://crates.io/crates/pallet-timestamp)<!-- markdown-link-check-disable-line -->
```

Comment:
You can disable link checks for this domain globally i think. Sucks that crates.io blocks it 😠

Reply:
Tried few times, always with no success. Also i dont get why the link gives a 404.even thought it is there. This was my only solution